.github/workflows/nightly_wan.yml

```diff
@@ -1,13 +1,17 @@
 name: Nightly -- Full WAN Network Tests
 
 on:
-# todo: this is totally broken atm. Fix and re-enable.
-#   schedule:
-#     - cron: "0 0 * * *"
+  schedule:
+    - cron: "0 0 * * *"
+# enable as below for testing purpose.
+  # pull_request:
+  #   branches: ["*"]
   workflow_dispatch:
 
 env:
   CARGO_INCREMENTAL: 0 # bookkeeping for incremental builds has overhead, not useful in CI.
+  DEFAULT_PAYMENT_FORWARD_SK: 49113d2083f57a976076adbe85decb75115820de1e6e74b47e0429338cef124a
+  NETWORK_NAME: DEV-01
```

Comment:
We might have to keep a unique `NETWORK_NAME` per `job`.
All the jobs inside the file will run in parallel and if you were to re-use the same testnet-name, we'd have a mess.